vueapp/src/components/TilePad/EditTileDialog.vue

```diff
@@ -55,7 +55,7 @@
 									{{ $t('editMode.tile.navigation') }}
 								</p>
 								<v-select
-									:disabled=disabled
+									:disabled="valid"
```

Comment:
Out of curiosity, what does this change do?

Reply:
It was a bug, it's supposed to be in quotation marks and the boolean valid is used to ensure it works in that way.